py/test/unit/selenium/webdriver/remote/remote_connection_tests.py

```diff
@@ -296,19 +307,42 @@ def test_register_extra_headers(mock_request, remote_connection):
     assert headers["Foo"] == "bar"
 
 
+def test_get_connection_manager_with_timeout_from_client_config():
+    client_config = ClientConfig("http://remote", timeout=300)
+    remote_connection = RemoteConnection(None, client_config=client_config)
+    conn = remote_connection._get_connection_manager()
+    assert conn.connection_pool_kw["timeout"] == 300
+    assert isinstance(conn, urllib3.PoolManager)
+
+
+def test_get_connection_manager_with_ca_certs_from_client_config():
+    client_config = ClientConfig("http://remote", ca_certs="/path/to/cacert.pem")
+    remote_connection = RemoteConnection(None, client_config=client_config)
+    conn = remote_connection._get_connection_manager()
+    assert conn.connection_pool_kw["timeout"] is None
+    assert conn.connection_pool_kw["cert_reqs"] == "CERT_REQUIRED"
+    assert conn.connection_pool_kw["ca_certs"] == "/path/to/cacert.pem"
+    assert isinstance(conn, urllib3.PoolManager)
+
+
 def test_get_connection_manager_ignores_certificates(monkeypatch):
```

Comment:
```suggestion
def test_get_connection_manager_ignores_certificates():
```

Minor nit: we can remove `monkeypatch` from params if no longer used

Reply:
Thanks! I updated it.